src/ExtensionsSample/Samples/MiscellaneousSamples.cs

```diff
@@ -13,14 +13,15 @@ public static class MiscellaneousSamples
         /// <summary>
         /// Demonstrates use of the ExecutionContext binding
         /// </summary>
-        [NoAutomaticTrigger]
-        public static void ExecutionContext(
-            ExecutionContext context,
-            TextWriter log)
-        {
-            string msg = string.Format("Invocation ID: {0}", context.InvocationId);
-            Console.WriteLine(msg);
-            log.WriteLine(msg);
-        }
+        // Re-add this when we reference the latest SDK version.
```

Comment:
The sample project will, unfortunately, be in a bad state until we're done migrating all of the extensions.

Reply:
Can't you wait until the core SDK change is made then pull it in? That seems like the correct staging.

Reply:
The core SDK changes are in. If we consume them here, the other extensions (that have not been migrated yet) will need to be disabled until they're migrated. So we'd have to comment all but the migrated ones, what I have now is the opposite.

Is that what you mean?